Node Status Monitor/NodeJS/api/auth.js

```diff
@@ -0,0 +1,50 @@
+import fetch from 'node-fetch';
+import config from '../config/config.js';
+import log from '../utils/logger.js';
+
+let token_url = config.token_url;
```

Comment:
If `token_url` is never changed, use `const`

Reply:
done